README.md

```diff
@@ -39,3 +39,69 @@ Most probably, you will have to adjust these four files:
 * log4j2.properties
 
 Setting up a Kitodo instance can be quite tricky. For more help on how to configure Kitodo, please check the [installation guides](https://github.com/kitodo/kitodo-production/wiki/Installationsanleitung), the [GitHub Wiki](https://github.com/kitodo/kitodo-production/wiki) or ask questions on the [mailing lists](https://github.com/kitodo/kitodo-production/wiki#Mailingliste).
+
+### Generating Javadocs
+
+The generation of Javadocs is issued by calling 
```

Comment:
I think "is issued" is not correct here, rather use "triggered" or even "started"